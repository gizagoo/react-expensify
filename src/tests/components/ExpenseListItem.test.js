import React from 'react';
import { shallow } from 'enzyme';
import ExpenseListItem from '../../components/ExpenseListItem';
import expenses from '../fixtures/expenses';
 
test('Should render ExpenseListitem with expenses', () => {
    const expense = expenses[1];
    const wrapper = shallow(<ExpenseListItem ExpenseListItem key={expense.id} {...expense} />);
    expect(wrapper).toMatchSnapshot();
});
