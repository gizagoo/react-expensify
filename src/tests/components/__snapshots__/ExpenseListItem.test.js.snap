// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render ExpenseListitem with expenses 1`] = `
<div>
  <h3>
    <Link
      replace={false}
      to="/edit/2"
    >
      Rent
    </Link>
  </h3>
  <p>
    109500
     - 
    -345600000
  </p>
</div>
`;
